src/scipp/spatial/__init__.py

```diff
@@ -199,17 +300,60 @@ def linear_transforms(*,
     """Constructs a :class:`Variable` with given dimensions holding an array
     of 3x3 matrices.
 
-    :seealso: :py:func:`scipp.matrix`
-
-    :param dims: Dimension labels.
-    :param values: Initial values.
-    :param unit: Optional, data unit. Default=dimensionless
+    Parameters
+    ----------
+    dims:
+        Dimension labels.
+    values:
+        Initial values.
+    unit:
+        Optional, data unit. Default=dimensionless
+
+    Returns
+    -------
+    :
+        An array variable of dtype ``linear_transform3``.
+    """
+    return _call_cpp_func(_core_cpp.matrices,
+                          dims=dims,
+                          unit=unit,
+                          values=_to_eigen_layout(values))
+
+
+def inverse(var: Variable) -> Variable:
```

Comment:
Have you considered naming? Numpy has it in a submodule (https://numpy.org/doc/stable/reference/generated/numpy.linalg.inv.html) and calls it `inv`. I don't think a submodule is what we want, but I thought I should bring it up anyway.

Is it worth adding a method as well, or is it too specialized?

Reply:
We already have it in a submodule. Our `spatial` somewhat corresponds to `numpy.linalg`. But I am fine with renaming it to `inv`. 

I wouldn't add a method because it only works on a handful of dtypes.